charts/rancher-monitoring/rancher-monitoring/charts/Chart.yaml

```diff
@@ -43,4 +99,5 @@ name: rancher-monitoring
 sources:
 - https://github.com/prometheus-community/helm-charts
 - https://github.com/prometheus-operator/kube-prometheus
-version: 9.4.204-rc09
+type: application
+version: 14.5.001-rc00
```

Comment:
Is this going to be a leading zeros issue?

Reply:
Ah yes. I'll follow what you did and patch this to 14.5.100-rc00

Reply:
addressed in latest commit